cds/config.py

```diff
@@ -198,7 +198,7 @@ def _(x):
     ),
     recid_embed=dict(
         pid_type='recid',
-        route='/record/<pid_value>/embed/<filename>',
+        route='/record/<pid_value>/embed',
```

Comment:
I have to double check, but probably you don't need to remove the `filename` from here